4.DZ/posts.hpp

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+
+namespace DZ5 {
+
+enum ETYPE {
+	LABORER,
+	SECRETARY,
+	MANAGER,
+	ACCOUNTANT,
+	EXECUTIVE,
+	RESEARCHER
+};
+
+void run() {
+	char letter;
+	ETYPE post;
+	std::cout << "Enter the first letter of the post: " << std::endl;
+	std::cin >> letter;
+	if (letter == 'L' || 'l') {
+		std::cout << "Full name of the post: laborer" << std::endl;
+	} else if (letter == 'S' || 's') {
+		std::cout << "Full name of the post: secretary" << std::endl;
+	} else if (letter == 'M' || 'm') {
+		std::cout << "Full name of the post: manager" << std::endl;
+	} else if (letter == 'A' || 'a') {
+		std::cout << "Full name of the post: accountant" << std::endl;
+	} else if (letter == 'E' || 'a') {
+		std::cout << "Full name of the post: executive" << std::endl;
+	} else if (letter == 'R'|| 'r') {
+		std::cout << "Full name of the post: researcher" << std::endl;
+	} else {
+		std::cout << "Post not found." << std::endl;
+	}
+	/* switch (letter) {
```

Comment:
Remove commented code, or save it somewhere.

Reply:
commented code as the second variant of writing code for this task